nav2_costmap_2d/include/nav2_costmap_2d/costmap_2d_publisher.hpp

```diff
@@ -136,6 +139,8 @@ class Costmap2DPublisher
   void prepareGrid();
   void prepareCostmap();
 
+  std::unique_ptr<nav2_msgs::msg::CostmapUpdate> get_raw_costmap_update_msg();
```

Comment:
It is better to use Camel Case, as shown below.  

```suggestion
  std::unique_ptr<nav2_msgs::msg::CostmapUpdate> getRawCostmapUpdateMsg();
```